source/dev/contribution_guide.md

```diff
@@ -61,7 +61,9 @@ We present the `Sphinx` docstring format [here](code_documentation.md). It is wo
 
 
 ## Communication
-Use [GitHub] to report bugs and propose features. Also, do not hesitate to contact us on our Slack workspace or by email at <modelinfo@breakthroughenergy.org> if you should have any questions.
+Use [GitHub] to report bugs and propose features. Also, do not hesitate to `Sign up <http://science.breakthroughenergy.org#get-updates>`_ to our email list and Slack workspace to stay updated and contact us if you should have any questions.
```

Comment:
rst to md copy
Should be`[sign up](http://science.breakthroughenergy.org#get-updates)`

Reply:
Done